draft-eggert-bcp45bis.md

```diff
@@ -164,12 +164,29 @@ Inappropriate postings include:
 
 # Security Considerations
 
-The IETF Chair, the IETF Executive Director, or a sergeant-at-arms {{SAA-SOP}}
-appointed by the Chair is empowered to restrict posting by a person, or of a
-thread, when the content is inappropriate and represents a pattern of abuse
-{{SAA-UPC}}. They are encouraged to take into account the overall nature of the
-postings by an individual and whether particular postings are an aberration or
-typical. Complaints regarding their decisions should be referred to the IAB.
+A sergeant-at-arms (SAA) appointed by the IETF Chair is empowered to
```

Comment:
Talking about terminology: When I mentioned this term to someone non-US outside of the IETF, that person was actually quite surprised by this term, given the role is supposed to explicitly not let the list get into a situation where "arms" are needed. Maybe this term is not as clear as it should be?

Reply:
FWIW the term is well established not just in the US but in most of the commonwealth nations. See [wiki](https://en.wikipedia.org/wiki/Serjeant-at-arms#Specific_countries).

Reply:
And might still be confusing for all the other nations... just saying... don't have a problem myself.

Reply:
FWIW, I was also somewhat surprised by the term when I started in the IETF. I wonder if it would be helpful to add a variant of the first sentence from the wikipedia page (e.g., "A  sergeant-at-arms is an officer appointed by a deliberative body to keep order during its meetings.") to the document, maybe with a link to the wikipedia page?

Reply:
Yes, there were some/a few (?) people who raised that concern, however, at least one person also wanted an independent committee that is not appointed by the chair. So as long as the chair appoints the SAA, I don't see a big difference. I personally find it more important to make sure that everybody who serves as a SAA feels conformable with the position and for me that would also mean that they have someone they can approach and rely on in cause of problems, rather than having them taking full responsibility given this is a volunteer position.

Reply:
Sounds like a good idea to me!